test/cluster/mzcompose.py

```diff
@@ -3164,3 +3164,93 @@ def subscribe():
                     ), e
             for thread in threads:
                 assert not thread.is_alive(), f"Thread {thread.name} is still running"
+
+
+def workflow_test_refresh_mv_warmup_after_restart(
+    c: Composition, parser: WorkflowArgumentParser
+) -> None:
+    """
+    Test that bootstrapping selects an as_of for an MV dataflow in a way that allows it to warm up before its next
+    refresh.
+    """
+
+    with c.override(
+        Materialized(
+            additional_system_parameter_defaults={
+                "enable_refresh_every_mvs": "true",
+            },
+        ),
+        Testdrive(no_reset=True),
+    ):
+        c.down(destroy_volumes=True)
+
+        c.up("materialized")
+        c.up("clusterd1")
+
+        # Create a new cluster
+        c.sql("DROP CLUSTER IF EXISTS cluster1 CASCADE;")
+        c.sql(
+            "ALTER SYSTEM SET enable_unmanaged_cluster_replicas = true;",
+            port=6877,
+            user="mz_system",
+        )
+        c.sql(
+            """
+            CREATE CLUSTER cluster1 REPLICAS (replica1 (
+                STORAGECTL ADDRESSES ['clusterd1:2100'],
+                STORAGE ADDRESSES ['clusterd1:2103'],
+                COMPUTECTL ADDRESSES ['clusterd1:2101'],
+                COMPUTE ADDRESSES ['clusterd1:2102'],
+                WORKERS 1
+            ));
+            """
+        )
```

Comment:
Can we just use a managed cluster here?

Also, I think the `destroy_volumes=True` above makes the `DROP CLUSTER IF EXISTS` unnecessary.

Reply:
I think with the unmanaged cluster the test might test compute reconciliation rather than as-of bootstrapping. Because you are not restarting the unmanaged cluster's replica (`clusterd1`), so the dataflow just stays hydrated when you restart envd.

Reply:
Changed it, thx! (There are many old tests that use unmanaged clusters, and it seems I was looking at only those when I was trying to figure out how to do this.)